### PR TITLE
hv: coding style: refine cpu related function to one exit

### PR DESCRIPTION
**Pull request by lifeix**
2) Fix procedure has more than one exit point.

Tracked-On: #861
Signed-off-by: Li, Fei1 <fei1.li@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Ok to verify

**Comment by HuiHuangShi** [comment]
2136<br>No new violations to the coding guideline detected.

File: hypervisor/arch/x86/cpu.c [hunk 1]
```diff
@@ -187,15 +189,16 @@ void init_cpu_post(uint16_t pcpu_id)
 
 static uint16_t get_cpu_id_from_lapic_id(uint32_t lapic_id)
 {
-	uint16_t i;
+	uint16_t pcpu_id = INVALID_CPU_ID;
 
-	for (i = 0U; (i < phys_cpu_num) && (i < CONFIG_MAX_PCPU_NUM); i++) {
+	for (uint16_t i = 0U; (i < phys_cpu_num) && (i < CONFIG_MAX_PCPU_NUM); i++) {
 		if (per_cpu(lapic_id, i) == lapic_id) {
```
[the remark this answers is not in the document]
Comment (lifeix) [reply hypervisor/arch/x86/cpu.c]: done

**Comment by HuiHuangShi** [comment]
2136<br>No new violations to the coding guideline detected.

**Pull request merged**